.gitmodules

```diff
@@ -3,7 +3,7 @@
   #url = https://github.com/spack/spack
   #branch = develop
   url = https://github.com/jcsda/spack
-  branch = jcsda_emc_spack_stack
+  branch = JCSDA/spack@24657b6
```

Comment:
```suggestion
  branch = jcsda_emc_spack_stack
```

Reply:
@climbfuji No problem at all! I am happy to contribute and thank you again  for the opportunity.